scm/driver/gitlab/webhook.go

```diff
@@ -202,6 +215,65 @@ func converBranchHook(src *pushHook) *scm.BranchHook {
 	}
 }
 
+func convertCommentHook(src *commentHook) (*scm.IssueCommentHook) {
+	var issue scm.Issue 
+	namespace, _ := scm.Split(src.Project.PathWithNamespace)
+	
+	switch src.ObjectAttributes.NoteableType {
+	case "Commit", "Issue", "Snippet":
+		// no op
+	case "MergeRequest":
+		pr := scm.PullRequest{
+			Number:  src.MergeRequest.Iid,
+			Title:   src.MergeRequest.Title,
+			Body:    src.MergeRequest.Description,
+			Ref:     fmt.Sprintf("refs/merge-requests/%d/head", src.MergeRequest.Iid),
+			Source:  src.MergeRequest.SourceBranch,
+			Target:  src.MergeRequest.TargetBranch,
+			Link:    src.Project.WebURL,
+			Closed:  src.MergeRequest.State != "opened",
+			Merged:  src.MergeRequest.State == "merged",
+			Author:  *convertUser(&src.User),
+			Created: parseTimeString(src.MergeRequest.CreatedAt),
+			Updated: parseTimeString(src.MergeRequest.UpdatedAt),
```

Comment:
Is there a way to get label and commits here? or it probably have to be a separate call

Reply:
Added labels and last commit Sha in the PR struct.